AudioCodeFinal1

```diff
@@ -0,0 +1,385 @@
+#time for parsingTokens
+import time
+
+#find the sound folder
+setMediaPath()
+sound=makeSound("Airhorn.wav")
+soundLength=getLength(sound)
+
+#lists for parsing tokens
+notesCombined = [] 
+melody = []
+notesMelody = []
+newParse = []  
+
+#global location for build Sine wave for parsingToken and sineWav  
+length=1
+buildSin = makeEmptySoundBySeconds(length)
+
+#PARSING TOKEN WRITE SONG 5
```

Comment:
This could be a [doc-string](https://www.python.org/dev/peps/pep-0257/). Please use doc-strings where appropriate.